tyk-api/tyk_api.go

```diff
@@ -50,22 +50,44 @@ type TykAPI struct {
 	GatewayConfig         EndpointConfig
 	DashboardConfig       EndpointConfig
 	CustomDispatcher      func(target Endpoint, method string, usercode string, body io.Reader) ([]byte, int, error) `json:"-"`
-	CustomSuperDispatcher func(target Endpoint, method string, body io.Reader) ([]byte, int, error) `json:"-"`
+	CustomSuperDispatcher func(target Endpoint, method string, body io.Reader) ([]byte, int, error)                  `json:"-"`
 }
 
 // PortalDeveloper represents a portal developer
 type PortalDeveloper struct {
-	Id            bson.ObjectId     `bson:"_id,omitempty" json:"id"`
-	Email         string            `bson:"email" json:"email"`
-	Password      string            `bson:"password" json:"password"`
-	DateCreated   time.Time         `bson:"date_created" json:"date_created"`
-	InActive      bool              `bson:"inactive" json:"inactive"`
-	OrgId         string            `bson:"org_id" json:"org_id"`
-	ApiKeys       map[string]string `bson:"api_keys" json:"api_keys"`
-	Subscriptions map[string]string `bson:"subscriptions" json:"subscriptions"`
-	Fields        map[string]string `bson:"fields" json:"fields"`
-	Nonce         string            `bson:"nonce" json:"nonce"`
-	SSOKey        string            `bson:"sso_key" json:"sso_key"`
+	Id              bson.ObjectId                           `bson:"_id,omitempty" json:"id"`
```

Comment:
@furkansenharputlu hi, I see this id to be bson.ObjectId, so am wondering if this will be an issue when dashboard use an SQL database.